src/IPinfo.php

```diff
@@ -16,6 +16,1017 @@
  */
 class IPinfo
 {
+    const CONTINENTS_DEFAULT = [
```

Comment:
Pls let's put it in some other file, this ends up filling the main logic file with a huge amount of data that must be scrolled through